lib/valkyrie/persistence/fedora/metadata_adapter.rb

```diff
@@ -24,6 +24,10 @@ def persister
       Valkyrie::Persistence::Fedora::Persister.new(adapter: self)
     end
 
+    def adapter_id
```

Comment:
Maybe just id over adapter_id? Curious about opinions. metadata_adapter.adapter_id seems more verbose than necessary, but if it'd be unclear this is fine. I also wonder if it should return a Valkyrie::ID to make it clear it's not just a string.